src/makeCalculator.js

```diff
@@ -4,7 +4,23 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    operate(method, number) {
+      this.result = method(this.result, number);
+
+      return this;
+    },
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+    add: (a, b) => a + b,
+    subtract: (a, b) => a - b,
+    multiply: (a, b) => a * b,
+    divide: (a, b) => a / b,
```

Comment:
The `divide` method does not handle division by zero. Consider adding a check to prevent division by zero, which could lead to runtime errors.